Library/Homebrew/extend/os/mac/keg.rb

```diff
@@ -108,7 +108,7 @@ def prepare_debug_symbols
   # Needed to make symlink permissions consistent on macOS and Linux for
   # reproducible bottles.
   def consistent_reproducible_symlink_permissions!
-    find do |file|
+    path.find do |file|
```

Comment:
See prior art in https://github.com/Homebrew/brew/commit/adfd12cfd05e2950c84a435c9e0be6963da9d749#diff-86ec75a3d285fc2a3f3e537aa9ec82e30f367fcceb9533cc1e7ac8c357a45187R527